Sources/SwiftSDKGenerator/Artifacts/DownloadableArtifacts.swift

```diff
@@ -13,71 +13,88 @@
 import struct Foundation.URL
 import struct SystemPackage.FilePath
 
-/// SHA256 hashes of Swift packages for Linux known to the generator.
-private let knownLinuxSwiftVersions: [LinuxDistribution: [String: [Triple.CPU: String]]] = [
-  LinuxDistribution.ubuntu(.jammy): [
+/// Information about the OS for which the artifact is built, if it's downloaded as prebuilt.
+private enum ArtifactOS: Hashable {
+  init(_ tripleOS: Triple.OS, _ versions: VersionsConfiguration) {
+    switch tripleOS {
+    case .linux:
+      self = .linux(versions.linuxDistribution)
+    case .macosx, .darwin:
+      self = .macOS
+    }
+  }
+
+  case linux(LinuxDistribution)
+  case macOS
+
+  /// No specific OS required for this artifact, it's not binary and is distributed in the source form.
+  case source
```

Comment:
Is this used anywhere? If it's not used now and we don't have a corresponding return value on this case for `llvmBinaryURLSuffix`, would it make sense to remove this case?

Reply:
Good catch, I've cleaned that up.